core/templates/msg.html

```diff
@@ -0,0 +1,76 @@
+<link rel="stylesheet" type="text/css" href="/css/msg.css">
+<section>
+<section id="center_panel">
+	<p id="msg">Messages</p>
+<button id="add_new" onclick="newMsg()" >Add New</button>
+<!--
+<div class="message">
+<button class="exit">X</button>
+Message 1
+</div>
+
+<div class="message">
+<button class="exit">X</button>
+Message 2
+</div>
+
+<div class="message">
+Message 3
+</div>
+
+<div class="message">
+<button class="exit">X</button>
+Message 4
+</div>
+-->
+<div id="add_message"></div>
+
+<div>
+<% all_messages.rows.forEach(function(rows){%>
+<div class="message"> <%=rows.message_body %></div>
+<%});
+%>
+</div>
+
+<div>
+    <div id="message"></div>
+    <form action="">
+      <input id="m" autocomplete="off" /><button>Send</button>
+    </form>
+    <script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
+    <script src="http://code.jquery.com/jquery-1.11.1.js"></script>
```

Comment:
Since you are already including jQuery in the layout file, this will just overwrite the other one. You can delete this line and also move the socket.io dependency straight to the layout, since other people are going to be using it as well. Same goes for the chat HTML